label_studio/core/settings/base.py

```diff
@@ -300,12 +300,6 @@
         'DB': 0,
         'DEFAULT_TIMEOUT': 180,
     },
-    'low': {
```

Comment:
Why this part is removed? 
I thought job decorator parameter has a higher priority and can override default, isn't it?

Reply:
Reverted